### PR TITLE
fix(moss-tts-local): disable vocoder graphs on WSL DXG

### PR DESCRIPTION
**Pull request by z2z23n0**
## Motivation

MOSS-TTS vocoder graph capture can fail on the ROCm WSL/DXG path and leave the HIP context unusable. Cleanup may then abort the process before the existing eager fallback can run.

## Modifications

- Default the vocoder to eager mode when running with HIP through `/dev/dxg`.
- Keep the existing graph-enabled default on CUDA and native ROCm.
- Reject an explicit `cuda_graph: true` on WSL/DXG before loading the codec or creating a graph.

Explicit `cuda_graph: false` remains unchanged. This does not affect attention backend selection.

## Accuracy Test

- Unit tests: `11 passed`
- Pre-commit: passed
- Windows WSL2, Ubuntu 24.04, RX 7900 XTX (`gfx1100`): the eager vocoder path completed a real request with HTTP 200 and produced a valid, non-silent WAV.

## Benchmark & Profiling

Not run. Other platforms keep the existing graph-enabled path.

### CODE REVIEW

File: sglang_omni/models/moss_tts_local/config.py [hunk 1]
```diff
@@ -3,8 +3,10 @@
 
 from __future__ import annotations
 
+import os
 from typing import Any, ClassVar
 
+import torch
 from pydantic import Field
```
[the remark this answers is not in the document]
Comment (zijiecode) [reply sglang_omni/models/moss_tts_local/config.py:9]: As the bot suggested, the current import is unnecessary, and a import within _uses_rocm_wsl_dxg() should be a better option.

**Review by Ratish1 — APPROVED** [review]
LGTM once the bot's comment is addressed above, thanks for the fix @z2z23n0

File: sglang_omni/models/moss_tts_local/config.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (z2z23n0) [reply sglang_omni/models/moss_tts_local/config.py:9]: Good catch — fixed in ba63f55d. `torch` is now imported inside `_uses_rocm_wsl_dxg()`, with `ImportError` falling back to `False`, and I added a regression test.

**Comment by z2z23n0** [comment]
@Ratish1 the bot comment is fixed and the update is pushed. Could you take another look when you have a moment? Thanks!
